whisper/whisperv6/whisper.go

```diff
@@ -520,16 +520,23 @@ func (wh *Whisper) runMessageLoop(p *Peer, rw p2p.MsgReadWriter) error {
 				log.Warn("failed to decode envelopes, peer will be disconnected", "peer", p.peer.ID(), "err", err)
 				return errors.New("invalid envelopes")
 			}
+
+			var trouble error
```

Comment:
maybe have it to be `trouble := []error{}` and store the `n` first errors ? More than 1 error can happen in real life.

Reply:
i am reluctant to complicate the processing logic. instead, i have changed the function so that it will report all the errors by logging. i have even upgraded the log level from warning to error, because this kind of error should be exceptional.